src/tbb/misc.h

```diff
@@ -25,7 +25,7 @@
 #include "oneapi/tbb/info.h"
 #endif /*__TBB_ARENA_BINDING*/
 
-#if __linux__ || __FreeBSD__
+#if __unix__ || __FreeBSD__
```

Comment:
I think FreeBSD defines unix, i.e.
```suggestion
#if __unix__
```
Check other places as well

Reply:
FreeBSD does not define unix as far as I've just checked. On the other hand, unix defines linux.

Reply:
It seems I am outdated, on FreeBSD 11 with clang 3.8.0, I have the following output
```shell
$clang++ -dM -E -x c++ /dev/null | grep unix
#define __unix 1
#define __unix__ 1
#define unix 1
```
Was something changed in newer versions?

Reply:
In other words with the compiler macro __unix__, you'll capture any unix system including FreeBSD and QNX. However, only using the compiler macro __FreeBSD__ leaves out other unix systems like QNX.